mistralrs-core/src/pipeline/mod.rs

```diff
@@ -82,6 +109,37 @@ pub struct SimpleModelPaths<P> {
     lora_preload_adapter_info: Option<HashMap<String, (P, LoraConfig)>>,
 }
 
+impl<P> SimpleModelPaths<P> {
```

Comment:
Can we rename this to `LocalModelPaths` to emphasize its purpose?

Reply:
Done.